### PR TITLE
Vale: test with one file

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Test linter with just one file and one line edit

### CODE REVIEW

File: website/docs/docs/trusted-adapters.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Trusted adapters take part in the Trusted Adapter Program, including a commitmen
 
 Free and open-source tools for the data professional are increasingly abundant. This is by-and-large a *good thing*, however it requires due diligence that wasn't required in a paid-license, closed-source software world. As a user, there are questions to answer important before taking a dependency on an open-source project. The trusted adapter designation is meant to streamline this process for end users.
 
-### Trusted adapter specifications
+### Trusted Adapter Specifications
```
Comment (nghi-ly) [inline website/docs/docs/trusted-adapters.md:13]: love the Vale warning for line 12!

**Review by nghi-ly — COMMENTED** [review]
this is amazing, @mirnawong1 ! 

a couple of questions:

1. since this is just a warning not an error, should the CI check still show up as failed/red? for only warnings, i expected pass/green.  

2. i took a look at the Vale details log. i see that "reviewdog" errored out (screenshot below), which was unexpected to me. is this normal/expected behavior? 

Screenshot 2024-08-15 at 1 14 22 PM

**Comment by mirnawong1** [comment]
closing as this is a test pr and most of the testing is being done https://github.com/dbt-labs/docs.getdbt.com/pull/5934

File: website/docs/docs/trusted-adapters.md [hunk 2]
```diff
@@ -9,11 +9,19 @@ Trusted adapters take part in the Trusted Adapter Program, including a commitmen
...
 
 
+<!-- vale off -->
+
+## Header Test To Test COmments
+more text here...
+
+<!-- vale on -->
+
+
```
Comment (mirnawong1) [inline website/docs/docs/trusted-adapters.md:24]: ## Header Test To Test COmments

**Pull request closed without merging**